### DISCUSSION TITLE
Star or Favorite Log msg from deletion

### DISCUSSION THREAD

**Discussion by susielinphsa** [datalust/seq-tickets#1959] [Ideas]
Would like to propose a feature of starring some log message, so they don't get deleted when the retention period comes., in case we have to go back to it for investigation or troubleshooting.

**Comment by liammclennan** [comment]
Hi @susielinphsa 

Creating a permalink for the event will do this.

**Reply by susielinphsa** [reply]
I see, that's great. So just to clarify, if we create a permalink, it will not delete the log if it hits the retention limit.

**Reply by liammclennan** [reply]
That's right. Here is the detail https://docs.datalust.co/docs/retention-policies#excluding-events-from-processing
